site-manage/site-manage-tool/tool/src/bundle/sitesetupgeneric.properties

```diff
@@ -1267,3 +1267,15 @@ sinfo.lessonbuildersubnav.name=Lessons subpage navigation
 sinfo.lessonbuildersubnav.allowForSite=Enable <strong>Lessons subpage navigation</strong> in the left tool menu.
 sinfo.lessonbuildersubnav.confirmEnabled=You have enabled the Lessons subpage navigation for this site.
 sinfo.lessonbuildersubnav.enabled=Enabled for the site
+
+# table toolbar common messages
+view.label=View
+gen.viewing.phrase=Viewing {0} - {1} of {2} items
+gen.show=Show {0} items...
+# {0} is the pagesize
+gen.previous.pagesize = Previous {0}
+# {0} is the pagesize
+gen.next.pagesize = Next {0}
+gen.select.label = Choose page size
+gen.listnavselect = To operate the combo box, first press Alt+Down Arrow to open it, and then use the up and down arrow keys to scroll through the options.
```

Comment:
Orphaned messages in site-manage will be cleaned up in a follow up PR for SAK-40098